nix-community/nixops-libvirtd
# Poetry2nixify

I couldn't actually get it to work because my libvirtd wasn't easy to set up. Maybe someone else can test it. 

testing steps:

    clone, check out the PR
    nix-shell
    poetry install
    poetry shell
    nixops create -d test tests/functional/....nix
    nixops deploy -d test

... make sure it works :P
